docs/config/ssr-options.md

```diff
@@ -2,17 +2,17 @@
 
 ## ssr.external
 
-- **Type:** `string[]`
+- **Type:** `string[] | true`
 - **Related:** [SSR Externals](/guide/ssr#ssr-externals)
 
-Force externalize dependencies for SSR.
+Externalize dependencies for SSR. By default, all dependencies are externalized, except linked dependencies for HMR. This can opted-out by adding the dependency to this option, or set `true` to force externalize all dependencies.
```

Comment:
it took me a little while to understand the opt-out was referring to linked dependencies. I might reword this a bit:
```suggestion
Externalize the given packages and their transitive dependencies for SSR. `true` will include all packages. By default, all dependencies are externalized except for linked dependencies. This option takes priority over `ssr.noExternal`.
```